Makefile.defs

```diff
@@ -145,8 +145,8 @@ endif
 GO_BUILD_FLAGS += -ldflags '$(GO_BUILD_LDFLAGS) $(EXTRA_GO_BUILD_LDFLAGS)' -tags=$(call join-with-comma,$(GO_TAGS_FLAGS)) $(EXTRA_GO_BUILD_FLAGS)
 GO_TEST_FLAGS += -tags=$(call join-with-comma,$(GO_TAGS_FLAGS))
 
-ifeq ($(NOSTRIP),1)
-	GO_BUILD_FLAGS += -gcflags="all=-N -l"
+ifeq ($(DEBUGBUILD),1)
+    GO_BUILD_FLAGS += -gcflags="all=-N -l"
```

Comment:
https://www.youtube.com/watch?v=SsoOG6ZeyUI :wink:

Reply:
We seem to be using spaces in other `ifeq` blocks, so I thought I'd make it consistent :smiley: